authors/apps/core/utilities.py

```diff
@@ -0,0 +1,21 @@
+from django.template.defaultfilters import slugify
+
+
```

Comment:
I think should be in a class for future usage

Reply:
I usually use classes to group related methods but in this case, I think a function is what I needed. How do you see?